packages/kbn-ai-playground/components/chat_sidebar.tsx

```diff
@@ -5,43 +5,63 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 
-import { EuiFormRow, EuiSwitch } from '@elastic/eui';
+import { EuiFormRow, EuiSwitch, EuiText } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 
 import { InstructionsField } from './instructions_field';
+import { OpenAIKeyCallout } from './open_ai_key_callout';
+import { OpenAIKeyFlyOut } from './open_ai_key_flyout';
+import { isEmpty } from 'lodash';
 
 export const ChatSidebar: React.FC = () => {
-  const [prompt, setPrompt] = React.useState('');
-  const [isIncludeCitations, setIncludeCitations] = React.useState<boolean>(true);
+  const [prompt, setPrompt] = useState('');
+  const [isIncludeCitations, setIncludeCitations] = useState<boolean>(true);
+  const [isOpenAIFlyOutOpen, setIsOpenAIFlyOutOpen] = React.useState<boolean>(false);
+  const [openAIKey, setOpenAIKey] = useState<string>('');
+
+  const onCloseOpenAIFlyOut = () => {
+    setIsOpenAIFlyOutOpen(!isOpenAIFlyOutOpen);
+  };
 
   return (
     <>
-      <InstructionsField
-        label={i18n.translate('aiPlayground.sidebar.instructionsField.label', {
-          defaultMessage: 'Instructions',
-        })}
-        placeholder={i18n.translate('aiPlayground.sidebar.instructionsField.placeholder', {
-          defaultMessage: 'Replace me',
-        })}
-        helpText={i18n.translate('aiPlayground.sidebar.instructionsField.help', {
-          defaultMessage:
-            'This is the instruction or question you want the AI to respond to. Be clear and specific for the best results.',
-        })}
-        value={prompt}
-        onChange={setPrompt}
-      />
+      {isOpenAIFlyOutOpen ? (
+        <OpenAIKeyFlyOut onClose={onCloseOpenAIFlyOut} setOpenAIKey={setOpenAIKey} />
+      ) : (
+        <>
+          {!isEmpty(openAIKey) ? (
```

Comment:
Nitpick: could you please change condition just to improve readibility instead (not empty) ? not empty : not not empty use (is empty) ? empty : not empty